gems/aws-sdk-s3/lib/aws-sdk-s3/client.rb

```diff
@@ -359,10 +359,6 @@ class Client < Seahorse::Client::Base
     #     for all object operations. You must first enable accelerate for
     #     each bucket. [Go here for more information](http://docs.aws.amazon.com/AmazonS3/latest/dev/transfer-acceleration.html).
     #
-    #   @option options [Boolean] :use_dualstack_endpoint (false)
```

Comment:
This looks like a mistake, but I don't think it is. Running `rake build` should populate this and fips.